index.html

```diff
@@ -10,10 +10,145 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    
+    <!--==================== HEADER ====================-->
+<header class="header" id="header">
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <!--==================== NAVBAR ====================-->
+    <div>
+        <img
+            src="img/karma-logo.svg"
+            alt="Karma official logo"
+            class="header_logo"
+        />
+    </div>
 
+    <nav class="container">
+        <a class="nav_icon-text">Meet Karma</a>
+            
+        <a class="nav_icon">How it Works</a>
+
+        <a class="nav_icon">Store</a>
+
+        <a class="nav_icon">Blog</a>
+
+        <a class="nav_icon">Help</a>
+
+        <a class="nav_icon">Login</a>
+    </nav>
```

Comment:
It would be good to use a `ul` with `li` elements for each link. 
So it would go nav > ul > li > a